Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocEnumValue.css

```diff
@@ -3,76 +3,80 @@
 span.bocEnumValue,
 div.bocEnumValue.readOnly
 {
-  width: 15em;
-  position: relative;
-  display: inline-block;
-  vertical-align: middle;
-  overflow: hidden;
+  display: inline-flex;
+  width: var(--remotion-themed-control-width);
+  min-height: var(--remotion-themed-min-height);
 }
 
-.bocEnumValue.readOnly
+.bocEnumValue td
 {
-  /* Ensures that the outline is visible in Firefox. */
-  overflow: visible;
+  white-space: nowrap;
 }
 
-.bocEnumValue.disabled
+.bocEnumValue td label
 {
+  padding: var(--remotion-themed-padding);
 }
 
-span.bocEnumValue select
+div.bocEnumValue td,
+div.bocEnumValue td span
```

Comment:
please split this into two blocks and merge the td blocks. the elements are nested and there's no technical reason (I can see) why these style should always match. They just are the same, right now.